tests/e2e/config/playwright.config.js

```diff
@@ -0,0 +1,69 @@
+const { defineConfig, devices } = require( '@playwright/test' );
+const { url } = require( './default.json' );
+
+module.exports = defineConfig( {
+	testDir: '../specs',
+
+	/* Maximum time one test can run for. */
+	timeout: 120 * 1000,
+
+	expect: {
+		/**
+		 * Maximum time expect() should wait for the condition to be met.
```

Comment:
`Maximum time, in milliseconds, expect() should wait for the condition to be met.`